src/faq.md

```diff
@@ -447,6 +447,57 @@ The reference runner and several other CWL implementations support running
 those Docker format containers using the Singularity engine. Directly
 specifying a Singularity format container is not part of the CWL standards.
 
+## How does glob work when describing output values?
+The field outputBinding describes how to set the value of each output parameter. The 'glob' field specifies a pattern to find files/directories relative to the output directory. The pattern used for glob' must be either relative to the output directory, an absolute path to the output directory, or an absolute path to an input file.
+
+CWL uses the POSIX glob(3) pathname matching. Wildcards are allowed in the glob field and are useful when If you don’t know the exact name of a file or directory in advance. The wildcard characters can either be an asterisk `*`, a question mark matching pathnames in directories. `?` or a range `[]`.
```

Comment:
The wildcard characters can either be an asterisk `*`, a question mark matching pathnames in directories, `?`,  or a range `[]`.

i think this needs to be a comma

Reply:
https://man7.org/linux/man-pages/man7/glob.7.html  --> this is glob7 -- I think we use glob3 so I am not sure why ? and [] works since it looks like they are glob7 specific.  But this says single character.

Reply:
do you  know where you got this from? the ? and the range?

Reply:
? does look like it works for a single character.  Not sure about range, let me check.